node/pkg/p2p/p2p.go

```diff
@@ -397,7 +397,7 @@ func Run(params *RunParams) func(ctx context.Context) error {
 		}
 
 		// Set up the attestation channel. ////////////////////////////////////////////////////////////////////
-		if params.gossipAttestationSendC != nil || params.obsvRecvC != nil {
+		if params.gossipAttestationSendC != nil || params.obsvRecvC != nil || params.batchObsvRecvC != nil {
```

Comment:
Looks like the `if params.obsvRecvC != nil {` statement below should be updated as well.